recipe/meta.yaml

```diff
@@ -22,11 +24,9 @@ requirements:
     - toolchain
     - cython
     - numpy x.x
-    - numpy >=1.6
   run:
     - python
     - numpy x.x
-    - numpy >=1.6
```

Comment:
Shouldn't we have both `numpy x.x` and `numpy >=1.6` as [was discussed in the PR to staged recepiesl](https://github.com/conda-forge/staged-recipes/pull/2173#discussion_r95095784)?